### PR TITLE
Completed html-forms exercise. - Chris Mendoza

### PR DESCRIPTION
**Pull request by CTMendoza**
html-forms3

### CODE REVIEW

File: html-forms/index.html [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+      <input type="checkbox" name="sign" value="yes" id="check">
+      <label for="check">Sign me up!</label>
+      <div>
+        <label for="comments">Comments</label>
+      </div>
+      <textarea name="textcomments" id="comments" cols="30" rows="10"></textarea>
+      <div>
+        <input type="submit" id="finished">
+      </div>
+      <label for="finished"></label>
```
Comment (BrettAlbright) [inline html-forms/index.html]: you can remove this label, the submit input doesn't need one

**Review by BrettAlbright — CHANGES_REQUESTED** [review]
Tiny update!

File: html-forms/index.html (hunk 1 above)
Comment (CTMendoza) [reply html-forms/index.html]: pushed again with label removed for the submit button

**Review by BrettAlbright — APPROVED** [review]
👍🏼

**Pull request merged**